### DISCUSSION TITLE
please help

### DISCUSSION THREAD

**Discussion by Christianrunbruh** [3kh0/ext-remover#424] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i try using it with chrome os and it does not work please help i click disable i go to a website and it does not work @3kh0

**Comment by Aka-but-nice** [comment]
Wut, do you mean? New Point Blank?

**Reply by Christianrunbruh** [reply]
look the extensions re enable
[Screen recording 2023-01-24 3.35.41 PM.webm](https://user-images.githubusercontent.com/123257486/214436761-e0e286b3-afa4-4adf-9d8d-ec9767bc5d20.webm)

**Reply by Aka-but-nice** [reply]
> look the extensions re enable [Screen recording 2023-01-24 3.35.41 PM.webm](https://user-images.githubusercontent.com/123257486/214436761-e0e286b3-afa4-4adf-9d8d-ec9767bc5d20.webm)


Your obviously on 106, and higher.

**Reply by Christianrunbruh** [reply]
how to downgrade back

**Reply by obkduishinijf** [reply]
> how to downgrade back

probably don't want to, since it ruins your chances of getting another one if you get caught. IF you understand the consequences, look at code and scroll to downgrade

**Reply by Christianrunbruh** [reply]
wdym?

**Reply by Christianrunbruh** [reply]
i understand but how

**Reply by Christianrunbruh** [reply]
can you send vid on how to downgrade

**Comment by obkduishinijf** [comment, marked as answer]
what are you even talking about? you need to be specific what you're even talking about.

**Reply by Christianrunbruh** [reply]
the extensions turn back on securly ect. i disable and it does not work 
[Screen recording 2023-01-24 3.35.41 PM.webm](https://user-images.githubusercontent.com/123257486/214436619-68baeb03-87d8-4411-b3f4-7bab7849a319.webm)

**Comment by EmpoweredPhoenix** [comment]
i believe he means that he goes onto a website and the extensions goes back on? am i correct???

**Reply by Christianrunbruh** [reply]
yes you are correct it does not work the extensions turn back on im on build 1.09 and it dont work

**Reply by Christianrunbruh** [reply]
[Screen recording 2023-01-24 3.35.41 PM.webm](https://user-images.githubusercontent.com/123257486/214436665-5dedb0ba-6613-4617-b46f-7431b5133101.webm)
